app/services/moderator/banish_user.rb

```diff
@@ -1,17 +1,18 @@
 module Moderator
   class BanishUser < ManageActivityAndRoles
+    def self.call(admin:, user:)
```

Comment:
I just moved it up since it's somewhat customary to have class methods before any of the instance related things.